interop/tapir/src/main/scala/caliban/interop/tapir/ws/Protocol.scala

```diff
@@ -49,15 +50,20 @@ object Protocol {
       override def complete(id: String): GraphQLWSOutput =
         GraphQLWSOutput(Ops.Complete, Some(id), None)
 
-      override def error[E](id: Option[String], e: E): GraphQLWSOutput =
+      override def error[E](id: Option[String], e: E): GraphQLWSOutput = {
+        val outputId = id match {
+          case None    => Some(UUID.randomUUID().toString)
```

Comment:
Nit: It would be better to use the `zio.Random` construct

Reply:
@SvenW can you just update that one? Then it's good to go 🙏